src/main/webapp/app/pages/LevelOfEvidencePage.tsx

```diff
@@ -120,20 +121,56 @@ const LEVEL_FILE_NAME: { [key in Version]: string } = {
     'CDRH’s-Approach-to-Tumor-Profiling-Next-Generation-Sequencing-Tests',
 };
 
+const REDIRECTPAGE: { [key in Version]?: PAGE_ROUTE } = {
+  [Version.V2]: PAGE_ROUTE[Version.V2],
+  [Version.DX]: PAGE_ROUTE[Version.DX],
+  [Version.PX]: PAGE_ROUTE[Version.PX],
+  [Version.FDA_NGS]: PAGE_ROUTE[Version.FDA_NGS],
+};
+
 @inject('routing', 'windowStore')
 @observer
 export default class LevelOfEvidencePage extends React.Component<
   LevelOfEvidencePageProps,
   any
 > {
   @observable version: Version = Version.V2;
+  @observable activeKey: Version = Version.V2;
+  pathname: string = window.location.pathname;
 
   readonly reactions: IReactionDisposer[] = [];
 
+  redirectToNewPage = (version: Version) => {
+    // if page is v2 page and version is v2 do nothing
+    const newPath = REDIRECTPAGE[version.toUpperCase()];
+    if (newPath) {
+      this.props.routing.history.push(newPath);
+    }
+  };
+
   updateLocationHash = (newVersion: Version) => {
-    window.location.hash = QueryString.stringify({ version: newVersion });
+    this.redirectToNewPage(newVersion);
+    this.updateActiveKey(newVersion);
+
+    if (window.location.pathname === PAGE_ROUTE.V2) {
+      window.location.hash = QueryString.stringify({
+        version: newVersion.toUpperCase(),
+      });
+    }
```

Comment:
Logics need to be updated here. If redictToNewPage is true, the rest of the methods probably should not be called.

Reply:
The rest of the function is still necessary after redirectToNewPage is called. The active key still needs to be updated so the correct tab is displayed and the code in the if statement just re-adds the hash for the V2, FDA and AAC pages after the page refreshes.